ui/components/app/signature-request/signature-request-data/signature-request-data.js

```diff
@@ -12,14 +12,14 @@ import {
   toChecksumHexAddress,
 } from '../../../../../shared/modules/hexstring-utils';
 import Box from '../../../ui/box';
```

Comment:
Can you update this file to use the `Box` component from the component-library?

Reply:
👍
Will do.
Thanks for pointing out.